src/Tuttifrutti/Sentry.hs

```diff
@@ -23,12 +25,24 @@ instance FromJSON Config where
       <$> v .: "environment"
       <*> v .: "dsn"
 
+initSentry
+  :: String
+  -> [(String, String)]
+  -> String
+  -> SentryLevel
+  -> Text
+  -> [(Text, Value)]
+  -> IO ()
+initSentry dsn initialTags serviceName level logMessage extraTags = do
+  sentry <- Sentry.initRaven dsn (Sentry.tags initialTags) Sentry.sendRecord Sentry.stderrFallback
+  Sentry.register sentry serviceName level (Text.unpack logMessage) (Sentry.extra $ map (first Text.unpack) extraTags)
+
+disabledHandle :: SentryLevel -> Text -> [(Text, Value)] -> IO ()
```

Comment:
What is this for?

Reply:
It's for situations where we don't have Sentry configuration available, so it's basically an "empty" handle.